msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java

```diff
@@ -126,17 +126,17 @@ public UUID getCorrelationId() {
 
     @Nullable
     private UUID sanitizeCorrelationId(@Nullable final String correlationId) {
-        final String methodName = "sanitizeCorrelationId";
+        final String methodTag= TAG + ":sanitizeCorrelationId";
```

Comment:
nit: spacing

Reply:
here and few other places..
**methodTag =**

Reply:
Here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1612/commits/8c46290fffbcb6a6af9d3ae985e8a327198a686a